src/styles/components/header.scss

```diff
@@ -111,27 +111,29 @@ $content-width: 90%;
 
 .hero-header {
   @include box-shadow();
+  flex-direction: column;
   height: 60vh;
   margin-bottom: 2em;
-  display: grid;
 
-  &__title {
-    grid-column: 2/14;
-    text-align: center;
+  & > * {
```

Comment:
Actually this might not be needed...will remove is so

Reply:
Without this the input box shrank
<img width="1507" alt="Screenshot 2020-12-21 at 13 56 33" src="https://user-images.githubusercontent.com/4357962/102784536-74479080-4394-11eb-9f50-685892ffdf90.png">